src/constants.ts

```diff
@@ -15,25 +15,29 @@ import qualityAssuranceQuiz from "./data/quality-assurance-quiz";
 import securityQuiz from "./data/security-quiz";
 import sqlQuiz from "./data/sql-quiz";
 import regexQuiz from "./data/regex-quiz";
-
+const mapQuizCategory = (quiz: any[], category: string)=> {
+  return quiz.map((q) => ({ ...q, Category: category }));
+}
 export const ALL_CATEGORIES = [
-  ...accessibilityQuiz,
-  ...cssQuiz,
-  ...cloudComputingQuiz,
-  ...devopsQuiz,
-  ...freecodecampQuiz,
-  ...generalCSQuiz,
-  ...gitQuiz,
-  ...htmlQuiz,
-  ...informationTechnologyQuiz,
-  ...javascriptQuiz,
-  ...linuxQuiz,
-  ...pythonQuiz,
-  ...sqlQuiz,
-  ...agileQuiz,
-  ...qualityAssuranceQuiz,
-  ...securityQuiz,
-  ...regexQuiz
+  ...mapQuizCategory(accessibilityQuiz, "Accessibility"),
+  ...mapQuizCategory(cssQuiz, "CSS"),
+  ...mapQuizCategory(cloudComputingQuiz, "Cloud Computing"),
+  ...mapQuizCategory(devopsQuiz, "DevOps"),
+  ...mapQuizCategory(freecodecampQuiz, "FreeCodeCamp"),
```

Comment:
misspelling of freeCodeCamp

```suggestion
  ...mapQuizCategory(freecodecampQuiz, "freeCodeCamp"),
```

Reply:
<img src="https://media0.giphy.com/media/25aA1NOirpGqteJcDp/giphy.gif"/>